dbNSFP.pm

```diff
@@ -128,10 +128,19 @@ sub new {
     }
     $index++;
   }
-  print STDERR "Include SO ", join(' ', keys %INCLUDE_SO), "\n";
   
   # get dbNSFP file
   my $file = $self->params->[$index];
+  my $version;
+  if ($file =~ /2\.9/) {
```

Comment:
Could we use a more generic `if/else` ?
e.g.:
```
if ($version < 4.0) { 
  do this
}
else { 
  do that
}```

Reply:
I need to specifically catch version 2.9 and 4.0 because they differ in the way coords are represented. For 2.9 there is a dbNSFP file with only GRCh37 coords and 4.0 has both GRCh37 and GRCh38 coords but changed how they name GRCh38 since version 3.5. But it is correct that I also need to match 3.x files.